tests/baselines/reference/jsxEsprimaFbTestSuite.js

```diff
@@ -72,8 +72,8 @@ baz
 <div>@test content</div>;
 <div><br />7x invalid-js-identifier</div>;
 <LeftRight left={<a />} right={<b>monkeys /> gorillas</b> /  > }/>
-    < a.b > ;
-a.b > ;
+    ,

```

Comment:
interesting, I didn't realize that we print missing nodes. In this case I'm not sure that user would expect to see it - this is our internal recovery strategy and I don' think it should leak outside. Maybe we should not emit missing nodes (since they are  ...missed in the source code)